dozer-types/protos/cloud.proto

```diff
@@ -5,21 +5,12 @@ import "google/protobuf/timestamp.proto";
 
 service DozerCloud {
   rpc create_application(CreateAppRequest) returns (AppResponse);
```

Comment:
Do we want to also remove `create_application, `update_application` and `get_application`?

Reply:
Lets do this in another PR. As @karolisg is still figuring out best way to deal with secrets. Currently using secrets doesn't work as application is not created yet.